Mathlib/AlgebraicGeometry/EllipticCurve/Projective/Basic.lean

```diff
@@ -382,9 +382,7 @@ lemma nonsingular_smul (P : Fin 3 → R) {u : R} (hu : IsUnit u) :
     refine (nonsingular_iff P).mpr ⟨(equation_smul P hu).mp hP, ?_⟩
     contrapose! hP'
     simp only [smul_fin3_ext]
-    exact ⟨by linear_combination (norm := ring1) u ^ 2 * hP'.left,
```

Comment:
I'm on the fence as to whether this is a loss in readability or not. (Somebody familiar with the mathematics should say if this is a boring detail that should be golfed into the ground or not. I don't do algebraic geometry.)